src/vm/gcenv.ee.cpp

```diff
@@ -397,6 +397,16 @@ Thread* GCToEEInterface::GetThread()
     return ::GetThread();
 }
 
+void GCToEEInterface::YieldProcessorNormalized()
+{
+    ::YieldProcessorNormalized(YieldProcessorNormalizationInfo());
+}
+
+void GCToEEInterface::EnsureYieldProcessorNormalizedInitialized()
```

Comment:
How is this going to be used? The extra condition to verify that the counts were initialized is pretty small relative to the call overhead. I think you can just check for the initialization inside YieldProcessorNormalized. No need to have a special API for it.

Reply:
Even if you want to use an interface call per YieldProcessorNormalized(), this shouldn't be necessary. The first few calls (before initialization) will use default values and subsequent calls with use initialized values.